magda-typescript-common/src/JsonConnector.ts

```diff
@@ -321,7 +321,7 @@ export default class JsonConnector {
             });
         });
 
-        app.get("/v0/datasets/:id/publisher", (req, res) => {
+        app.get("/v0/datasets/:id/organisations", (req, res) => {
```

Comment:
I think we can leave this for now, given that we'd have to update admin-client too.